ydb/public/tools/lib/cmds/__init__.py

```diff
@@ -310,6 +311,50 @@ def enable_pqcd(arguments):
     return (getattr(arguments, 'enable_pqcd', False) or os.getenv('YDB_ENABLE_PQCD') == 'true')
 
 
+def merge_two_yaml_configs(data_1, data_2):
+    if not (isinstance(data_1, (dict, list)) or data_1 == None):
```

Comment:
it needs to check equal to None with is.
https://peps.python.org/pep-0008/#programming-recommendations